From: IME Packaging Team <pkg-ime-devel@lists.alioth.debian.org>
Date: Sun, 5 Feb 2012 21:12:25 +0800
Subject: 01-lower_sunpinyin_version.patch

===================================================================
---
 CMakeLists.txt |    2 +-
 src/eim.cpp    |    4 +++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ab4779..68f3a44 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,7 @@ FIND_PACKAGE(Libintl REQUIRED)
 FIND_PACKAGE(Gettext REQUIRED)
 FIND_PACKAGE(Fcitx 4.2.0 REQUIRED)
 
-PKG_CHECK_MODULES(SUNPINYIN REQUIRED "sunpinyin-2.0 >= 2.0.4" )
+PKG_CHECK_MODULES(SUNPINYIN REQUIRED "sunpinyin-2.0 >= 2.0.3" )
 
 # uninstall target
 configure_file(
diff --git a/src/eim.cpp b/src/eim.cpp
index d5790f9..605fc32 100644
--- a/src/eim.cpp
+++ b/src/eim.cpp
@@ -400,7 +400,8 @@ void FcitxSunpinyinDestroy (void* arg)
 
 INPUT_RETURN_VALUE FcitxSunpinyinDeleteCandidate (FcitxSunpinyin* sunpinyin, FcitxCandidateWord* candWord)
 {
-    if (candWord->owner == sunpinyin)
+/*
+	if (candWord->owner == sunpinyin)
     {
         CCandidateList pcl;
         sunpinyin->view->getCandidateList(pcl, 0, sunpinyin->candNum);
@@ -410,6 +411,7 @@ INPUT_RETURN_VALUE FcitxSunpinyinDeleteCandidate (FcitxSunpinyin* sunpinyin, Fci
         classicView->deleteCandidate(*index, mask);
         return IRV_DISPLAY_CANDWORDS;
     }
+*/
     return IRV_TO_PROCESS;
 }
 
-- 
