Index: fcitx-sunpinyin/CMakeLists.txt
===================================================================
--- fcitx-sunpinyin.orig/CMakeLists.txt	2012-02-05 21:09:54.293720968 +0800
+++ fcitx-sunpinyin/CMakeLists.txt	2012-02-05 21:09:58.501741835 +0800
@@ -10,7 +10,7 @@
 FIND_PACKAGE(Gettext REQUIRED)
 FIND_PACKAGE(Fcitx 4.2.0 REQUIRED)
 
-PKG_CHECK_MODULES(SUNPINYIN REQUIRED "sunpinyin-2.0 >= 2.0.4" )
+PKG_CHECK_MODULES(SUNPINYIN REQUIRED "sunpinyin-2.0 >= 2.0.3" )
 
 # uninstall target
 configure_file(
Index: fcitx-sunpinyin/src/eim.cpp
===================================================================
--- fcitx-sunpinyin.orig/src/eim.cpp	2012-02-05 21:09:54.309721039 +0800
+++ fcitx-sunpinyin/src/eim.cpp	2012-02-05 21:09:58.501741835 +0800
@@ -400,7 +400,8 @@
 
 INPUT_RETURN_VALUE FcitxSunpinyinDeleteCandidate (FcitxSunpinyin* sunpinyin, FcitxCandidateWord* candWord)
 {
-    if (candWord->owner == sunpinyin)
+/*
+	if (candWord->owner == sunpinyin)
     {
         CCandidateList pcl;
         sunpinyin->view->getCandidateList(pcl, 0, sunpinyin->candNum);
@@ -410,6 +411,7 @@
         classicView->deleteCandidate(*index, mask);
         return IRV_DISPLAY_CANDWORDS;
     }
+*/
     return IRV_TO_PROCESS;
 }
 
